ScrubBarTests/ScrubBarTests.swift

```diff
@@ -669,3 +856,29 @@ class AnimatingMock: Animating {
 func == (lhs: AnimatingMock.AnimationConfiguration, rhs: AnimatingMock.AnimationConfiguration) -> Bool {
     return String(describing: lhs) == String(describing: rhs)
 }
+
+extension ScrubBarItem: Equatable {}
+
+extension ScrubBarItem {
+
+    static func empty() -> ScrubBarItem {
```

Comment:
ScrubBarItem has reference semantics and because of that I deliberately use a function instead of a property. That helps to illustrate that `empty()` is not idempotent.